zerver/tornado/event_queue.py

```diff
@@ -841,7 +841,7 @@ def get_client_payload(apply_markdown: bool, client_gravatar: bool) -> Dict[str,
         if 'sender_delivery_email' not in dct:  # nocoverage
             dct['sender_delivery_email'] = dct['sender_email']
 
-        MessageDict.finalize_payload(dct, apply_markdown, client_gravatar)
+        dct = MessageDict.finalize_payload(dct, apply_markdown, client_gravatar)
```

Comment:
You can just do `return ...` here, no need to save it as `dct`. (See the next line).

Reply:
Okay